index.bs

```diff
@@ -715,8 +715,7 @@ keyword token is used, then the [=identifier=] of the operation argument
 is simply that token.
 
 The [=identifier=] of any of the abovementioned
-IDL constructs (except operation arguments) must not be "<code>constructor</code>",
-"<code>toString</code>",
+IDL constructs (except operation arguments) must not be "<code>toString</code>",
```

Comment:
Revert this, please. It forbids, for example, `const constructor = 7;`, which clashes with the `Interface.prototype.constructor` property.

Reply:
It is already forbidden, now, because constructor is a terminal.